scribble-lib/scribble/acmart/acmart.tex

```diff
@@ -4,8 +4,10 @@
 % These are replaced by scribble/acmart/style.tex,
 %  which is used in combination with acmart.cls
 
+\newcommand{\SAuthorinfo}[4]{#1}
```

Comment:
Yes, yes it does. All of them are used:

* Author Name
* Author Email
* Author Institute
* Author Orcid

Now, this wasn't in David's scribble/acmart because that version expected everyone to essentially write latex in scribble, where as this one actually follows the scribble pattern for doing things.

(Notice that the command is redefined in `acmart/style.tex`, which redefines all of those commands. Effectively, every command in `acmart/acmart.tex` is just a stub.